test/common.js

```diff
@@ -616,3 +616,18 @@ exports.WPT = {
     assert.fail(undefined, undefined, `Reached unreachable code: ${desc}`);
   }
 };
+
+// Useful for testing expected internal/error objects
+exports.expectsError = function expectsError(code, type, message) {
```

Comment:
This function was added to `common.js` but not documented in the `test/README.md`.